src/renderer/components/ObservationCard.tsx

```diff
@@ -49,7 +49,7 @@ export default function ObservationCard(props: ObservationCardProps) {
   ];
```

Comment:
Changing to default value by selecting the value in the dropdown menu doesn't work now:
![Peek 2022-08-16 12-34](https://user-images.githubusercontent.com/47003650/184859807-68029243-72e8-4bc8-87e3-bc43f2d5dd98.gif)

Reply:
Good catch! :+1: This problem was introduced in #298. I made some changes to your PR before merging as this functionality wasn't working correctly, but it turns out I didn't get it quite right :wink: I'll add it to my notes and create a separate issue to fix.